sdk/storage/azure-storage-blobs/inc/azure/storage/blobs/blob_options.hpp

```diff
@@ -146,21 +135,6 @@ namespace Azure { namespace Storage { namespace Blobs {
      */
     Azure::Core::Nullable<std::string> EncryptionScope;
 
-    /**
-     * @brief Specify the number of retries and other retry-related options.
-     */
-    StorageRetryWithSecondaryOptions RetryOptions;
```

Comment:
So we decided to drop the support for retry to secondary

Reply:
No, it just need to be re implemented as a policy after the core retry policy which will check the request the retry number for the that request to decide if switching to secondary